feature-detects/css/animations.js

```diff
@@ -8,7 +8,7 @@
   "warnings": ["Android < 4 will pass this test, but can only animate a single property at a time"],
   "notes": [{
     "name" : "Article: 'Dispelling the Android CSS animation myths'",
-    "href": "http://goo.gl/OGw5Gm"
+    "href": "https://daneden.me/2011/12/14/putting-up-with-androids-bullshit/"
```

Comment:
Oh were you saying the google+ link didn't work? here's a screenshot:

<img width="560" alt="screen shot 2016-01-15 at 13 10 11" src="https://cloud.githubusercontent.com/assets/143402/12343601/6aaaeda8-bb89-11e5-8769-8d33a7d22c9a.png">